rtl/include/eclhelper.hpp

```diff
@@ -44,8 +44,8 @@ typedef unsigned short UChar;
 
 //Should be incremented whenever the virtuals in the context or a helper are changed, so
 //that a work unit can't be rerun.  Try as hard as possible to retain compatibility.
-#define ACTIVITY_INTERFACE_VERSION      650
-#define MIN_ACTIVITY_INTERFACE_VERSION  650             //minimum value that is compatible with current interface
+#define ACTIVITY_INTERFACE_VERSION      651
```

Comment:
@ghalliday this seems like overkill when the only way ECL can access the new function is through a new plugin method.  But a workunit that uses the new plugin feature will not be backward compatible with 7.0.x.

Is there a better way?

Reply:
I don't think we change the interface version for cases where new functions have been added and only new code uses it.